jsapp/js/components/submissions/submissionDataList.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import bem, {makeBem} from 'js/bem'
+import {
+  getFlatQuestionsList,
+  FlatQuestion
+} from 'js/assetUtils'
+import {getRowData} from 'js/components/submissions/submissionUtils'
+import './submissionDataList.scss'
+
+bem.SubmissionDataList = makeBem(null, 'submission-data-list', 'ul')
+bem.SubmissionDataListQuestion = makeBem(null, 'submission-data-list-question', 'li')
+bem.SubmissionDataListQuestion__path = makeBem(bem.SubmissionDataListQuestion, 'path')
+bem.SubmissionDataListQuestion__label = makeBem(bem.SubmissionDataListQuestion, 'label', 'h3')
+bem.SubmissionDataListQuestion__response = makeBem(bem.SubmissionDataListQuestion, 'response')
+
+type SubmissionDataListProps = {
+  assetContent: AssetContent
+  submissionData: SubmissionResponse
+  /** Whether to include meta type questions in the rendered list. */
+  includeMeta?: boolean
```

Comment:
I'm not sure how this would get populated as the `submissionDataDetails` seems to be only sending `assetContent` and `submissionData`.

Reply:
It's removed in #3488 nevermind!